kin-sdk-core/src/main/java/kin/sdk/core/Request.java

```diff
@@ -0,0 +1,79 @@
+package kin.sdk.core;
+
+
+import android.os.Handler;
+import android.os.Looper;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+public class Request<T> {
+
+    private static final ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
@berryve we can change to a newSingleThreadExecutor like we said at the meeting?
in order to avoid the multithreaded behavior.

Reply:
changed to single thread executor, all requests will run sequentially